benchmarks/playfair_cipher/oop_c#/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Text;
+using System.Drawing;
+using System.Text.RegularExpressions;
+using System.IO;
+
+namespace oop_c_
+{
+    class Program
+    {
+        static string TEST_STRING = File.ReadAllText("benchmarks/playfair_cipher/lines.txt");
+        static void Main(string[] args)
+        {
+            PlayFairCipher p = new PlayFairCipher("this is a great keyword");
+            string encrypt = p.Encrypt(TEST_STRING);
+            System.Console.WriteLine(encrypt.Length);
+            string decrypt = p.Decrypt(encrypt);
+            System.Console.WriteLine(decrypt.Length);
+        }
+    }
+
+    public class Table
+    {
+        private char[,] charTable;
+        private Point[] positions;
+        private Point currentPos;
+        public Table()
+        {
+            charTable = new char[5, 5];
+            positions = new Point[26];
+            currentPos = new Point();
+        }
+
+        public void AddNext(char c)
+        {
+            charTable[currentPos.X, currentPos.Y] = c;
+            positions[c - 'A'] = currentPos;

```

Comment:
Kunne man overveje noget operatoroverloading til opslag i positions?

Reply:
Ville sikkert være nice, men jeg ved ikke hvordan man gør.

Reply:
@lrecht you go